grafana_client/api.py

```diff
@@ -42,6 +42,7 @@
     AsyncDatasource,
     AsyncFolder,
     AsyncHealth,
+    AsyncLibraryElement,
```

Comment:
It looks like this was missing, right? Thanks for spotting, and fixing per 4144e92!

Maybe we can come up with an idea and routine to prevent such details slipping through on future occasions, on behalf of `script/generate_async.py`, if it's not too much of a hassle?

/cc @Ousret

Reply:
It should be easy to implement, yes.
There's another option, a bit more straightforward (without generate_async.py): "getattr + module async inspect" on init to check if one api is missing in both sync and async. then raise a warning or error.

regards,